example/lambdas/s3AccessTest/index.js

```diff
@@ -91,7 +91,7 @@ async function handler(event) {
     'write-object': testWrite,
     'list-objects': testList,
   };
-  return testChoices[testName](s3, params);
+  return await testChoices[testName](s3, params);
```

Comment:
Just to make sure I'm understanding, this is redundant, right? We want to always require `await` but in cases like this we don't actually expect it to function differently. Is that right?

Reply:
So long as the thing we are calling returns a Promise, that is correct.